app/views/course/achievements/_form.html.slim

```diff
@@ -7,4 +7,30 @@
 
   = f.input :published
 
+  - if @achievement && !@achievement.new_record?
+    h2
+      span = t('course.condition.header')
+      div.dropdown.pull-right
+        button.btn.btn-primary.dropdown-toggle#dropdownBtn [data-toggle="dropdown"
```

Comment:
Seems like the backslash only works for Ruby code. Breaking the line in the middle of the attribute list is the only configuration I could get to work.

Maybe I could align the items better, so it's clearer that it's an attribute list?